rfcs/0012-gossip.md

```diff
@@ -0,0 +1,48 @@
+# Gossip, A.K.A. Cluster Info
+
+This RFC describes the Solana's gossip protocol, also known as the network control plane.
+
+Nodes continuously share signed data objects among themselves in order to find each other, structure the network data plane, prevent censoring, etc.
+
+The basic mechanism is a 10Hz loop wherein every node sends a push message and/or a pull message.  Push and pull messages may elicit responses, and push messages may be forwarded on to others in the network.
+
+Gossip runs on a well-known UDP/IP port, or a port in a well-known range.  Once a network is bootstrapped, nodes advertise to each other where to find their gossip endpoint (a socket address).
+
+## Gossip Records
+
+Records shared over gossip are arbitrary, but signed and versioned (with a timestamp), and need to make sense to the node receiving them.  A strategy of "most recently recevied wins" is employed to manage what's stored in cluster info.
```

Comment:
@rob-solana Its most recent timestamp which is signed by the creator, not most recently received.